mac/Keyman4MacIM/Keyman4MacIM/KMDownloadKeyboard/KMDownloadKBWindowController.m

```diff
@@ -23,7 +23,11 @@ - (void)windowDidLoad {
     [super windowDidLoad];
     
     [self.webView setFrameLoadDelegate:(id<WebFrameLoadDelegate>)self];
-    [self.webView.mainFrame loadRequest:[NSURLRequest requestWithURL:[NSURL URLWithString:@"https://secure.tavultesoft.com/prog/macosx/10/downloadkeyboards/"]]];
+    
+    NSString *version = [[[NSBundle mainBundle] infoDictionary] objectForKey:@"CFBundleShortVersionString"];
+    NSString *url = [NSString stringWithFormat:@"https://keyman.com/go/macos/10.0/download-keyboards/?version=%@", version];
```

Comment:
Good question. In some ways it'd probably be sensible to extract it from the major version of the product but it's not hugely important. It can be helpful to not have to retool all the API endpoints when they don't change between versions (creates a bunch of busywork!)

Reply:
Makes sense